### DISCUSSION TITLE
Some suggestions from my experience

### DISCUSSION THREAD

**Discussion by rik1599** [swayos/swayos.github.io#32] [Ideas]
I loved SwayOS since the first time I installed it on my Debian 12. I think it is a very good project, but I think that some decision made by the devs are so weird because I noticed that the script installs a lot of redundant programs (for not to say bloatware).
As a lover of minimal setups I forked the repository and I'm trying to "clean" the installation script (in my case for debian)
[https://github.com/rik1599/swayos.github.io/blob/main/setup-debian.sh](url). 

Some major changes I made until now:
- substituted emacs-nox with nano (but also vim is a viable alternative)
- substituted wofi with tofi (as suggested [here](https://github.com/swayos/swayos.github.io/issues/12)). Tofi uses the fullscreen theme (see [here](https://github.com/philj56/tofi?tab=readme-ov-file#theming))
- added jq (used in wcp script but not preinstalled by default)
- removed completely pactl in favor of wpctl (Wireplumber) both in sway [config](https://github.com/rik1599/swayos.github.io/blob/main/home/.config/sway/config) and in wcp [script](https://github.com/milgra/wcp/pull/10)
- added mod4 + l shortcut for swaylock
- fix for xdg-desktop-portal-wlr already implemented in sway config
I hope I have been helpful

**Comment by prirai** [comment]
@rik1599 I think the repository is inaccessible or perhaps you deleted it. Could you somehow share it as I was going in a similar direction with Debian.
